File: app/src/main/java/com/musicplayer/SocyMusic/utils/UiUtils.java
package com.musicplayer.SocyMusic.utils;

import android.content.Context;
import android.util.TypedValue;

public class UiUtils {
    public static int dpToPixel(Context context, int dp) {
        return (int) TypedValue.applyDimension(
                TypedValue.COMPLEX_UNIT_DIP,
                dp,
                context.getResources().getDisplayMetrics());
    }
}

File: app/src/main/java/com/musicplayer/OpenMusic/utils/UiUtils.kt
package com.musicplayer.OpenMusic.utils

import android.content.Context
import android.util.TypedValue

object UiUtils {
    @JvmStatic
    fun dpToPixel(context: Context, dp: Int): Int {
        return TypedValue.applyDimension(
            TypedValue.COMPLEX_UNIT_DIP,
            dp.toFloat(),
            context.resources.displayMetrics
        ).toInt()
    }
}